riff-raff/build.sbt

```diff
@@ -6,13 +6,15 @@ resolvers ++= Seq(
     "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
 )
 
+ivyScala := ivyScala.value map { _.copy(overrideScalaVersion = true) }
```

Comment:
A quick Google suggests it's down to https://github.com/guardian/riff-raff/blob/sh-bump-casbah/build.sbt#L3

We should try bumping that to 2.11.7 and getting rid of this.